src/gfx_func.h

```diff
@@ -21,7 +21,7 @@
  * (this is btw. also possible if needed). This is used to avoid a
  * flickering of the screen by the video driver constantly repainting it.
  *
- * This whole mechanism is controlled by an rectangle defined in #_invalid_rect. This
+ * This whole mechanism was controlled by an rectangle defined in #_invalid_rect. This
```

Comment:
I replaced it with a lovely story that described exactly how the dirty block system worked in my previous pull request; just haven't gotten so far as to do that here. It will happen :wink:.